lib/scrabble.rb

```diff
@@ -1,4 +1,63 @@
+require 'pry'
+
 class Scrabble
+  attr_reader :record
+  
+  def initialize
+    @record = {}
+  end
+
+  def add_play(player_id, word)
+    if !@record[player_id]
+      @record[player_id] = []
+    end
+    @record[player_id] << word
+  end
+
+  def score(string, multi = nil, double = nil)
```

Comment:
This method is pretty long! When methods start getting this long, see if you can make the required method `score` into a CEO-type method which calls a few helper methods.